motors_roboteq_canopen.orogen

```diff
@@ -2,10 +2,13 @@
 
 name "motors_roboteq_canopen"
 
+cxx_standard "c++17"
```

Comment:
FYI, you shouldn't need this if the dependencies (in your case, the motors_roboteq_canopen driver library) is using C++17 already.